### ISSUE TITLE
BlueMap causes clients to think they need it locally.

### ISSUE THREAD

**Issue by Graftax** [BlueMap-Minecraft/BlueMap#232]
### What i did / Steps to reproduce
1. Read the BlueMap CurseForge page and note that it says "This is a SERVER-ONLY mod!".
2. Shut down server and add BlueMap-1.5.5-forge-1.16.4.jar to the server mods folder, then start it again.
3. Open Minecraft and note the red "X" in the server selection menu indicating that mods are out of sync.
4. Open logs/latest.log and note the line: "Server has additional mods that may be needed on the client: bluemap@1.5.5".

### Expected result
Client is not warned about mod sync when BlueMap is present on the server.

### Actual result
Clients are warned that they may be out of sync when selecting the server. 

### Context
BlueMap-1.5.5-forge-1.16.4
Linux/Forge 36.2.0
Had BlueMap-1.6.0-forge-1.16.4 installed and downgraded to 1.5.5. Clients are still able to connect, but the sync issue causes them to report it as a problem to me, repeatedly.  

A minor problem, hopefully the fix is as easy as updating `acceptableRemoteVersions`.

**Comment by LiskoSlayer63** [comment]
>A minor problem, hopefully the fix is as easy as updating `acceptableRemoteVersions`.

From Forge Documentation:
```
//Make sure the mod being absent on the other network side does not cause the client to display the server as incompatible
ModLoadingContext.get().registerExtensionPoint(ExtensionPoint.DISPLAYTEST, () -> Pair.of(() -> FMLNetworkConstants.IGNORESERVERONLY, (a, b) -> true));```

**Comment by TBlueF** [comment]
https://mcforge.readthedocs.io/en/1.17.x/concepts/sides/#writing-one-sided-mods

**Comment by TBlueF** [comment]
Should be fixed in 1.6.1
